rust/src/lib/nm/query_apply/vpn.rs

```diff
@@ -66,6 +66,9 @@ fn get_libreswan_conf(nm_set_vpn: &NmSettingVpn) -> LibreswanConfig {
         ret.dpdaction = data.get("dpdaction").cloned();
         ret.ipsec_interface = data.get("ipsec-interface").cloned();
         ret.authby = data.get("authby").cloned();
+        ret.leftmodecfgclient =
+            data.get("leftmodecfgclient").map(|s| s == "yes");
```

Comment:
This would be easier to understand if it did not use a single letter variable. I do not have any idea what `s` means that makes it a boolean.

Reply:
single level variable is widely used in rust in side of `map()`, `and_then` and etc.